examples/AdvancedOperations/AddAdCustomizer.php

```diff
@@ -372,21 +372,21 @@ private function createFeedItems(
                 $result->getResourceName(),
                 PHP_EOL
             );
-            $feedItemResourceNames []= $result->getResourceName();
+            $feedItemResourceNames[] = $result->getResourceName();
         }
 
         return $feedItemResourceNames;
     }
 
     /**
-     * Helper function to create a FeedItemOperation.
+     * Creates a FeedItemOperation.
      *
-     * @param string name the value of the Name attribute.
-     * @param string price the value of the Price attribute.
-     * @param string date the value of the Date attribute.
-     * @param feedResourceName the resource name of the feed.
-     * @param feedAttributes the attributes to be set on the feed.
-     * @return a FeedItemOperation to create a feed item.
+     * @param string name the value of the Name attribute
+     * @param string price the value of the Price attribute
+     * @param string date the value of the Date attribute
+     * @param string adCustomizerFeedResourceName the resource name of the feed
+     * @param array adCustomizerFeedAttributes the attributes to be set on the feed
+     * @return FeedItemOperation to create a feed item
```

Comment:
FeedItemOperation the feed item operation to create a feed item.

(You need to repeat it)

Reply:
Done.